### PR TITLE
Make waving less obtrusive

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by penguinland — APPROVED** [review]
:star_struck: LGTM! Thanks for such a fast turnaround.

**Comment by JessamyT** [comment]
Thanks for writing code that is easy to understand such that this was a trivial change :D

File: robot.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ async def _wiggle_on_inactivity(self):
             while True:
                 await asyncio.sleep(self.INACTIVITY_PERIOD_S)
                 self._logger.debug("wiggle wiggle wiggle")
-                for _ in range(3):
+                for _ in range(1):
                     await self._servo.move(self.UPPER_POSITION +
                                            self.WIGGLE_AMOUNT)
                     await asyncio.sleep(self.WIGGLE_DELAY_S)
```
Comment (penguinland) [inline robot.py:109]: I can't comment on line 113, but I'd say "wiggling" instead of "wiggle." While we're changing this file.

**Review by purplenicole730 — APPROVED** [review]

File: robot.py (hunk 1 above)
Comment (JessamyT) [reply robot.py:109]: Done!

**Pull request merged**